md/release-notes.md

```diff
@@ -1,44 +1,38 @@
 # Release notes
 
-Bonita BPM 7.5 includes a set of new features related to the Living Application development and deployment and UI personalization in addition to important technology updates and bug fixes.
+Bonita BPM 7.5 includes a set of new features related to Living Application development and deployment as well as UI personalization, in addition to important technology updates and bug fixes.
 
 <a id="living-application-development-and-deployment"/>
 
-## Living Application Development and Deployment
-
-### Integration of SSO SAML 2.0
-Bonita BPM works with Single-Sign-On (SSO) solutions using SAML V2 protocol `(Teamwork, Efficiency, Performance edition)` : 
-* Being able to use standard authentication service 
-* Can connect to Bonita apps with my company credentials (per tenant configuration)
-* Common SAML 2.0 Identity providers : ADFS (MS Active Directory Federation Services), SalesForce, Open AM, Google Auth,...
+## Living Application development and deployment
 
 ### New wizard to import a process in the studio & compare the artifacts 
-Bonita BPM Studio provides a wizard when importing a .bos that display the duplicate artifacts (forms, connectors, ...)  and ask for actions to the users. Usual actions are proposed to user for every artifacts : keep existing or overwrite with the new import :
-* Ease collaboration between teams who works with .bos exchange
-* Ease the collaboration by file exchange and reduce drastically the risk of overwrite error during import of a .bos.
+Bonita BPM Studio provides a wizard when importing a .bos that displays the duplicate artifacts (forms, connectors, ...)  and asks for user actions, between keeping the existing artifact and overwriting with the newly imported artifact:
```

Comment:
Still 7.5 and no need of a variable since 7.6 Release note will be a different page.